### PR TITLE
add ckvignesh

### LINKED ISSUE #825: Vignesh CK
Github Page

### PR DESCRIPTION
**Pull request by ckvignesh**
resolve #825

### CODE REVIEW

**Comment by ckvignesh** [comment]
Hi @eddiejaoude, can you please help with this?

**Comment by schmelto** [comment]
Closing in favor of #826

File: data.json [hunk 1]
```diff
@@ -1552,5 +1552,11 @@
       "githubUsername": "metantonio",
       "imageUrl": "https://user-images.githubusercontent.com/51391473/139906982-fb25c6da-4255-41b6-9f53-71f48f78d32b.png",
       "issueNumber": 820
+},
+  {
```
Comment (schmelto) [inline data.json:1556]:
```suggestion
  },
  {
```

File: data.json [hunk 2]
```diff
@@ -1552,5 +1552,11 @@
...
+      "name": "Vignesh CK",
+      "githubUsername": "ckvignesh",
+      "imageUrl": "https://user-images.githubusercontent.com/61013244/140084080-ba40ae9a-c470-4a82-99a4-42dbbffca978.png",
+      "issueNumber": 825
 }
```
Comment (schmelto) [inline data.json:1561]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (ckvignesh) [reply data.json]: Done, @schmelto ...

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — COMMENTED** [review]
LGTM! 🚀

**Review by krishguptadev — COMMENTED** [review]
LGTM!

**Comment by ckvignesh** [comment]
> LGTM! rocket

Thank you!

**Comment by ckvignesh** [comment]
> LGTM!

Thank you!

**Pull request merged**